runtime/vm/mz64/UpcallThunkGen.cpp

```diff
@@ -35,16 +35,207 @@ extern "C" {
 
 #if JAVA_SPEC_VERSION >= 16
 
+// constants to map specific register instances.
+#define R1 1
+#define R2 2
+#define R4 4 // R4 = CallerSP
+#define R6 6 // entry point
+#define R7 7 // return address
+#define R15 15
+
+static I_32
+BC(I_8 *instructionPtr, I_32 mask1, I_32 offsetRegister, I_32 destinationRegister, I_32 displacementInteger)
+{
+   I_32 instructionSize = 4;
```

Comment:
Is there specific reason for instructionSize variable in this and all subsequent instruction functions? Should it simply return instruction size directly ?

Reply:
We can use #define for the constants here as well. Do you prefer that or should we just return the literal constant like below for each method ?

```
return 4;
```

Reply:
Yes, literal constant should be fine. We do not need to use #define here.

Reply:
Should be fixed in [968f54e](https://github.com/eclipse-openj9/openj9/pull/19701/commits/968f54e09c9943b50bacbc4329189a3ef85ec21f)